frontend/android/src/main/res/menu/menu_nav_drawer.xml

```diff
@@ -13,7 +13,7 @@
     </group>
     <group
         android:id="@+id/features"
-        android:checkableBehavior="none"
+        android:checkableBehavior="single"
```

Comment:
I thought it's better to add selected color to every drawer menu.

<img width="468" alt="20190513_060340_Pixel 3" src="https://user-images.githubusercontent.com/1269214/57587743-dc8d8980-7544-11e9-8f3f-a2c9071b1da9.png">